icubam/predicu/tests/test_load_public_data.py

```diff
@@ -0,0 +1,14 @@
+import datetime
+
+import predicu.data
+
+
+def test_load_public_data():
+  data = predicu.data.load_public_data()
```

Comment:
I'm wondering if we should skip this by default. And have some pytest flag to run it optionally.

Reply:
yep let's do that!